CDTDatastore/query/CDTQQuerySqlTranslator.m

```diff
@@ -99,7 +99,7 @@ + (CDTQQueryNode *)translateQuery:(NSDictionary *)query
 
         if (allDocsIndex.length > 0) {
             NSString *tableName = [CDTQIndexManager tableNameForIndex:allDocsIndex];
-            NSString *sql = [NSString stringWithFormat:@"SELECT _id FROM %@;", tableName];
+            NSString *sql = [NSString stringWithFormat:@"SELECT _id FROM '%@';", tableName];
```

Comment:
Is there a reason why you're using `'%@'` here instead of `\"%@\"` as you are doing elsewhere?